.gitlab-ci.yml

```diff
@@ -93,6 +93,8 @@ hotfix_job:
     - $CI_PROJECT_DIR/ci/regress.sh "check_design.syn TB=bp_processor CFG=e_bp_single_core_cfg" bp_top &
     - $CI_PROJECT_DIR/ci/regress.sh "check_design.syn TB=bp_processor CFG=e_bp_quad_core_cfg" bp_top &
     - $CI_PROJECT_DIR/ci/regress.sh "check_design.syn TB=bp_softcore" bp_top &
+    - $CI_PROJECT_DIR/ci/regress.sh "check_design.syn TB=bp_processor CFG=e_bp_single_core_ucode_cce_cfg" bp_top &
+    - $CI_PROJECT_DIR/ci/regress.sh "check_design.syn TB=bp_processor CFG=e_bp_quad_core_ucode_cce_cfg" bp_top &
```

Comment:
Similar to checking the design with both FSM and ucode CCE, we likely need to enforce regressions being run against both designs, otherwise the non-default (ucode CCE) will almost certainly encounter bugs due to not being in the regression, even though we want to maintain it as supported. The question is do we create separate jobs for testing configs with FSM or ucode CCE, or do we test both configs within the same job. Properly testing both designs will effectively double our regressions, but the other option is to insufficiently test.

Reply:
How about just doubling the check_design targets and mc_* regressions?  I doubt we'll find any issues in the single core tests that we don't find there

Reply:
I think separate jobs is fine.  I'm going to refactor our CI system when I get some time in the next few weeks, so it won't be a high overhead